spanner/session.go

```diff
@@ -1067,6 +1067,7 @@ func (p *sessionPool) close(ctx context.Context) {
 	}
 	p.mu.Unlock()
 	p.hc.close()
+	close(p.multiplexedSessionReq)
```

Comment:
This change would get lost if we remove the entire `sessionPool` implementation (which is the intention). So it might be better to put this line of code in the `Client#Close()` function instead to make sure it is not unintentionally deleted. AFAICT, there are no tests that verify that this is actually closed, meaning that it would also not be caught by any tests.